### PR TITLE
Added Deals in Hero Section as  Slider

### PR DESCRIPTION
**Pull request by Nitheesha33**
## Type of PR
- [ ] Bug fix
- [X] Feature enhancement
- [ ] Documentation update
- [ ] Other (specify): ___________  

## Description
This pull request aims to add a new slider in the hero section of the website to showcase ongoing and upcoming deals, as well as new arrivals. The slider enhances the user experience by allowing customers to quickly view the latest offers and arrivals in a visually appealing manner.

## Screenshots / Videos (if applicable)
**Before:**

https://github.com/user-attachments/assets/1cdb549e-afa4-494a-b808-361cb940bc87

**After:**

https://github.com/user-attachments/assets/fb0679aa-79cb-45aa-9d7f-948fd7b824ac

## Checklist
- [X] I have performed a self-review of my code.
- [X] I have tested the changes thoroughly before submitting this pull request.
- [X] I have provided relevant issue numbers, screenshots, and videos after making the changes.
- [X] I have commented my code, particularly in hard-to-understand areas.

### Additional Context
I have implemented three slides in total: one for ongoing deals, one for upcoming deals, and another for new arrivals. The challenge was to ensure smooth transitions between the slides while maintaining performance. After considering different design options, I decided to prioritize a clean, minimalist layout for better readability and engagement.

Thank you for reviewing my pull request!

@swaraj-das Please label my issue with the appropriate level and then merge my pr

### CODE REVIEW

**Comment by Nitheesha33** [comment]
@swaraj-das the only issue is with a PR without the label. Other than that, my PR is perfectly able to deploy. I request you to look into this asap.

File: hero-slider/effects.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
+let slideIndex = 1;
```
Comment (swaraj-das) [inline hero-slider/effects.js:1]: Do not create a separate 'hero-slider' folder. Instead, please add the JavaScript for the hero slider directly into the existing script.js file.

File: index.html [hunk 2]
```diff
@@ -52,6 +52,17 @@
...
+                    <li>
+                        <div class="searchbar-outer">
+                            <div class="search-bar hidden" id="searchBar">
+                                <input type="text" id="search-input" placeholder="Search..." />
+                                <button type="button" onclick="performSearch()"><i class="fas fa-search"></i></button>
+                            </div>
+                        </div>
+                    </li>
+
                     <li class="login"><a href="login/login.html">Login</a></li>
```
Comment (swaraj-das) [inline index.html:66]: Add two buttons to the slider that allow users to navigate the images left (previous) and right (next) within the slider.

File: index.html [hunk 3]
```diff
@@ -52,6 +52,17 @@
                     <li><a href="#keyboard">Gaming Keyboard</a></li>
                     <li><a href="#others">Others</a></li>
 
+                    <!--adding short search bar-->
```
Comment (swaraj-das) [inline index.html:55]: You can position the search bar at the bottom of the navbar to enhance the user experience.

**Pull request closed without merging**